requirements/constraints.txt

```diff
@@ -93,7 +93,7 @@ libsass==0.10.0
 click==8.1.6
 
 # pinning this version to avoid updates while the library is being developed
-openedx-learning==0.11.4
+# openedx-learning==0.11.5
```

Comment:
@rpenido Uncomment this line

Reply:
Fixed: b8a74a796a41ff0dcc40e5ed6ae04425f9937ac4